DolbyIO/Source/Private/DolbyIOSubsystem.cpp

```diff
@@ -310,6 +314,10 @@ void UDolbyIOSubsystem::Connect(const FString& ConferenceName, const FString& Us
 	UserInfo.name = ToStdString(UserName);
 	UserInfo.externalId = ToStdString(ExternalID);
 	UserInfo.avatarUrl = ToStdString(AvatarURL);
+	{
+		FScopeLock Lock{&RemoteParticipantsLock};
+		RemoteParticipants.Reset();
```

Comment:
This should also be in DemoConference. It probably even deserves its own method at that point.